class AuthError(Exception):
    pass


class EntityError(Exception):
    pass
